index.js

```diff
@@ -79,18 +79,14 @@ module.exports = {
     "no-trailing-spaces": 1,
     "no-useless-constructor": 1,
     "one-var": [ 1, { "uninitialized": "always", "initialized": "never" } ],
-
-    /* "object-curly-spacing": [ 2, "never" ], */
-
+    "object-curly-spacing": [ 1, "never" ],
```

Comment:
was always the style, just not possible to enforce with the previous paren spacing rule